day_7/fizzbuzz.rb

```diff
@@ -0,0 +1,18 @@
+def FizzBuzz (x,y)
+  while x <= y
```

Comment:
When there are assumptions around numbers, like `y > x`, using variable names like `max` and `min` are extremely helpful.

Reply:
I understand and appreciate the feedback.